### ISSUE TITLE
TypeError while using AdaptiveTreeRegressor() 

### ISSUE THREAD

**Issue by debanshu08** [online-ml/river#1115]
I am trying to have a regression model using HoeffdingAdaptiveTreeRegressor(). My target column is an integer prediction 
and my input features are a mixture of string and int values
x = {'A': 'AZ', 'INV': '24000834793', 'Days': 21, 'Type': 'DRY', 'Size': 20, 'Utility': 'Plastic', 'SPOT': 'N'}
y = 1 

**_Getting TypeError: can't multiply sequence by non-int of type 'float'_**
on running 
for x, y in stream.iter_pandas(pdDf, target):
    print(x)
    print(y)
    y_pred = model.predict_one(x)

Basically predict_one is throwing this TypeError error, can someone help with this?

_Originally posted by @debanshu08 in https://github.com/online-ml/river/discussions/1114_

**Comment by debanshu08** [comment]
Got fixed Thanks
